src/Quote/quote.js

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from 'react';
+import './quote.css';
+
+export default function ShowQuote() {
+  const [retell, setRetell] = useState([]);
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      const category = 'amazing';
+      const apiKey = 'bnqnnOM1gde1qM8Farv5hA==Z9wth7aFcidEsatZ';
+      try {
+        const res = await fetch(
+          `https://api.api-ninjas.com/v1/quotes?category=${category}`,
+          {
+            methode: 'GET',
+            headers: {
+              'X-Api-Key': apiKey,
+            },
+          },
+        );
+
+        const json = await res.json();
+        setRetell(json[0]);
+      } catch (error) {
+        setHasError(true);
+      }
+      setIsLoading(false);
+    };
+    fetchData();
+  }, []);
+
```

Comment:
- [ ] I see your array is empty, it needs to have setIsLoading and setRetell so that when they change, the state too will change.